README.md

```diff
@@ -1 +1,105 @@
-# crescendo
\ No newline at end of file
+<<<<<<< Updated upstream
+<<<<<<< Updated upstream
+# crescendo
+=======
+=======
+
+>>>>>>> Stashed changes
+# crescendo
+
+Software versions and requirements:
+
+If you are on Mac, install XCode from the mac app store, git comes with it. WPI Lib installer requires Xcode
+
+WPI Lib version 2024.1.1 
+https://github.com/wpilibsuite/allwpilib/releases/tag/v2024.1.1
+
+WPI Lib Documentation 2024
+https://docs.wpilib.org/en/stable/ 
+
+Git (Github works with git, this is command line tool used to push code and other functions of github)
+https://git-scm.com/downloads   (any version is okay)
+
+GitHub Desktop (newest version is fine)
+https://desktop.github.com/
+
+---------------------------------------------------------------------
+
+<<<<<<< Updated upstream
+
+
+=======
+>>>>>>> Stashed changes
+Install WPI Lib
+
+- ensure Xcode is installed if you're on a mac
+
+<<<<<<< Updated upstream
+- visit https://github.com/wpilibsuite/allwpilib/releases/tag/v2024.1.1 and download version 2024.1.1
+
+- open the setup executable
+
+=======
+- visit https://github.com/wpilibsuite/allwpilib/releases/tag/v2024.1.1 (2.0GB) and download version 2024.1.1
+
+- open the iso / disk image
```

Comment:
There was a conflict here and I wasnt sure if you wanted it to say open iso or setup if this isnt what you want let me know